### ISSUE TITLE
Add On-Hover animation on handlers 

### ISSUE THREAD

**Issue by Snailedlt** [SplitScreen-Me/splitscreenme-hub#31]
Add on-hover animations to the handlers, like we have on the handlers here: https://www.splitscreen.me/

CSS
```css
.handlers {
  transition: transform .2s; /* Animation */
  margin: 0 auto;
}

.handlers:hover {
  transform: scale(1.1); /* (110% zoom - Note: if the zoom is too large, it will go outside of the viewport) */
}
```

**Comment by Lord-Baldwin-of-Kerry** [comment]
Hi, I would love to handle this.
In the repository I only see 2 less files, and I notice the most components are styled from antd. And the custom styles are done trough inline styling. Just want to make sure that it is OK to create new styling files in order to use the native CSS hover selector.

**Comment by Lord-Baldwin-of-Kerry** [comment]
@Snailedlt  Also, how can I see the handlers on the dev local build? the only way to login right now is with an email but I can't do the email verification because the client id is missing in the configuration. Any help will be greatly appreciated.

**Comment by Snailedlt** [comment]
Hi @Lord-Baldwin-of-Kerry , I'm glad to see more people joining this project.

> Hi, I would love to handle this.
> In the repository I only see 2 less files, and I notice the most components are styled from antd. And the custom styles are done trough inline styling. Just want to make sure that it is OK to create new styling files in order to use the native CSS hover selector.

I'm not sure if that's the way to do it, it was just an example of how it could be done, and it's the same way we do it in the splitscreenme-www project. That said, if you could do it with antd, that would probably be best. @r-mach would be the best person to ask these questions.

> @Snailedlt Also, how can I see the handlers on the dev local build? the only way to login right now is with an email but I can't do the email verification because the client id is missing in the configuration. Any help will be greatly appreciated.

I've sent a message to @r-mach about this, hopefully he gets back to me, or responds here shortly :)

**Comment by r-mach** [comment]
We need to create some fixtures so new people have a set of data to play with.

I believe creating .less files is the way to go. The preprocessor should works. I will try to take a look on what's the best practice to follow (obviously inline styling in React components isn't)

**Comment by Snailedlt** [comment]
> We need to create some fixtures so new people have a set of data to play with.
> 
> I believe creating .less files is the way to go. The preprocessor should work. I will try to take a look on what's the best practice to follow (obviously inline styling in React components isn't)

On the note of making it more accessible for new people. Should we also add node, python and npm to dependencies? Is it possible to add them to the project dependencies somehow?

**Comment by r-mach** [comment]
@Lord-Baldwin-of-Kerry 

I added a `theme.less` file in client folder. This one can be used for creating classes as of now. We will split it eventually and use import.

For the fixture part, as of now, maybe @Snailedlt could help you getting prod db loaded in your local environment as he did it many times and wrote a tutorial :)

**Comment by Snailedlt** [comment]
Oh gosh, no pressure 😅

**Comment by Lord-Baldwin-of-Kerry** [comment]
@Snailedlt where could I get a hold of this tutorial? and I'll start working on the hover animation!

**Comment by Snailedlt** [comment]
@Lord-Baldwin-of-Kerry First you need to download NoSQLBooster. 
Before you do the next steps, run the project like normal with `npm run dev`
Then from there do these steps:

1. Get dump from database: 
 create a folder on your desktop (or anywhere else) called `dump`. Inside `dump`, create a folder called `dbutils`
 Download this [database dump file](https://easyupload.io/uf83or), and put it inside the dump folder. pw is `hub_dump` 

2. reimport
 download MongoDB database tools from here: https://www.mongodb.com/try/download/database-tools
 put them in the `dbutils` folder
In the terminal: 
 `cd dbutils`
 `./mongorestore --host=localhost:3001 --gzip --archive=../hub_prod_db.gz --nsFrom="hubdb.*" --nsTo="hubdb.*"`

3. Connect to db in noSQLBooster
 connect to localhost:3001
 drop "meteor" database
 rename "hubdb" to "meteor"


**If this doesn't work for you, add me on discord: `Snailed lt#3563`, so we can figure it out in dms**

**Comment by Snailedlt** [comment]
Forgot to close this issue.
Fixed in #44
